docs/skills/ecommerce_bot_skill.rst

```diff
@@ -1,7 +1,7 @@
 eCommerce Bot
```

Comment:
If it is a page for the agent we should consider adding a new section to the documentation or rename current 'Skills' to 'Skills & Agents'.
@yoptar @seliverstov @litinsky